.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -17,6 +17,7 @@
   - These should run automatically, but if not, you can manually trigger them locally using `python scripts/pre_commit_linter.py` and `bash scripts/run_frontend_tests.sh`.
 - [ ] The PR is made from a branch that's **not** called "develop".
 - [ ] The PR follows the [style guide](https://github.com/oppia/oppia/wiki/Coding-style-guide).
+- [ ] Checked the codeowners for the files/folders changed and also carried out the corresponding checks for the files as laid out in the [codeowner's wiki](https://github.com/oppia/oppia/wiki/Oppia%27s-code-owners-and-checks-to-be-carried-out-by-developers).
```

Comment:
The PR addresses the points mentioned in the codeowner checks for the files/folders changed. (See the [codeowner's wiki page](...).)

Reply:
Made the change. PTAL!
Thanks!